When adding the environment variable JAVA_TOOL_OPTIONS an extra line
in the output from 'java -version' is produced. As this output is
parsed by configure script the extra line has to be filtered out.

--- jdk8u-4be07cb28b21.orig/common/autoconf/boot-jdk.m4
+++ jdk8u-4be07cb28b21/common/autoconf/boot-jdk.m4
@@ -51,7 +51,7 @@ AC_DEFUN([BOOTJDK_DO_CHECK],
             BOOT_JDK_FOUND=no
           else
             # Oh, this is looking good! We probably have found a proper JDK. Is it the correct version?
-            BOOT_JDK_VERSION=`"$BOOT_JDK/bin/java" -version 2>&1 | head -n 1`
+            BOOT_JDK_VERSION=`"$BOOT_JDK/bin/java" -version 2>&1 | grep -v JAVA_TOOL_OPTIONS | head -n 1`
 
             # Extra M4 quote needed to protect [] in grep expression.
             [FOUND_VERSION_78=`echo $BOOT_JDK_VERSION | grep  '\"1\.[78]\.'`]
